Changes

```diff
@@ -9,6 +9,9 @@ Working version
 
 ### Standard library:
 
+* #10859: Add `Format.pp_print_array`
```

Comment:
The `*` should be a `-` (it's not a breaking change, by our OCaml definition of breaking!)